nss-day-cohort-44/rare-rest-api-malicous-vortices
# Ts view user profiles

# View All User Profiles
## https://github.com/nss-day-cohort-44/rare-rest-malicious-vortices-client/issues/26


## Type of change

# Testing Instructions
`git fetch origin ts-view-user-profiles`
`git checkout ts-view-user-profiles` 

User Instructions: 
To test via Postman, use the following URL:
`http://localhost:8000/users`

This should return a list of all users IF your token is set up for an admin account. Otherwise you should see a message saying that you don't have admin privileges. 

# Checklist:
